lib/extension.js

```diff
@@ -6,6 +6,7 @@ var isFunction = require('min-dash').isFunction,
 
 var WILDCARD = '*';
 
+const zeebeServiceTaskProperties = ['zeebe:TaskDefinition', 'zeebe:Subscription', 'zeebe:TaskHeaders','zeebe:LoopCharacteristics'];
```

Comment:
Why is it only these four properties, and not also zeebe:Input, zeebe:Output, ...?

Reply:
Those are all the tasks that become available when an event becomes a zeebeServiceTask. The output, for example, was already available in any event so we don’t want to remove it in case it’s not a message event

Reply:
Okay, for output I get it, but what about for instance Input? Input is available for ZeebeServiceTasks, but shall not be available to NoneEndEvents or ThrowIntermediateEvents?

Reply:
Ok, I think I missed Input. I review the properties and double check which ones apply for ZeebeServiceTasks but not events